smithy-model/src/test/java/software/amazon/smithy/model/loader/ModelAssemblerTest.java

```diff
@@ -88,6 +92,26 @@ public void addsExplicitShapes() {
         assertThat(result.unwrap().getShape(ShapeId.from("ns.foo#Bar")), is(Optional.of(shape)));
     }
 
+    @Test
+    public void addsExplicitTraits() {
+        StringShape shape = StringShape.builder().id("ns.foo#Bar").build();
+
+        Node node = Node.fromStrings("validator");
+        TraitFactory provider = TraitFactory.createServiceFactory();
```

Comment:
A simpler test case could just directly create a SuppressTrait rather than needing to use a TraitFactory. Seems like the TraitFactory bit isn't under test here.

Reply:
Fixed.